bdb_transaction_cli/cli.py

```diff
@@ -1,15 +1,132 @@
-# -*- coding: utf-8 -*-
-
 import click
+import json
+
+from bigchaindb_common.crypto import generate_key_pair
+from bigchaindb_common.transaction import Transaction, \
+    Condition, Fulfillment, Metadata, Asset, Ed25519Fulfillment
+
+from bdb_transaction_cli.utils import json_argument, json_option, listify
+
+
+@click.group()
+def main():
+    pass
+
+
+@main.command()
+@click.option('--name', required=False, help=(
+    "Print the keys as shell variables,                             eg: "
+    "`export $(bdb generate_keys --name=bob)`"))
+def generate_keys(name):
+    """
+    Generate Ed25519 key pair.
+
+    Generates a random Ed25519 key pair separated by a space character.
+    First value is the public key, second is the private key.
+    """
+    priv, pub = generate_key_pair()
+    fmt = '{pub} {priv}'
+    if name:
+        fmt = '{name}_pub={pub} {name}_priv={priv}'
```

Comment:
Was thinking something similar, but then I get the impression we're trying to do too much here. Why not just return a JSON dict; how often do we really think people are going to be using this to export signing and verifying keys as env variables?